src/wam2layers/tracking/backtrack.py

```diff
@@ -43,7 +103,8 @@ def output_path(date, config):
 def read_data_at_date(d):
     """Load input data for given date."""
     file = input_path(d, config)
-    return xr.open_dataset(file, cache=False)
+    da = xr.open_dataset(file, cache=False)
+    return reduce_domain(da, config)
```

Comment:
This is the best place to do it for now, I think. The advantage is that you never read the redundant data into memory.  If we ever switch to time-dependent tracking_regions (specified through .nc file) we might need to move it around a bit.